cargo test compilation errors

Recreation
docker run -it --rm -v/tmp/rust/kxkdb:/src -w /src rust cargo test

```
root@1380ef2dce5a:/src# cargo test
   Compiling ipc_examples v0.0.1 (/src/ipc_examples)
error[E0432]: unresolved import `async_trait`
 --> ipc_examples/examples/acceptor_auth.rs:8:5
  |
8 | use async_trait::async_trait;
  |     ^^^^^^^^^^^ use of undeclared crate or module `async_trait`

error: cannot determine resolution for the attribute macro `async_trait`
  --> ipc_examples/examples/acceptor_auth.rs:11:3
   |
11 | #[async_trait]
   |   ^^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error[E0706]: functions in traits cannot be declared `async`
  --> ipc_examples/examples/acceptor_auth.rs:13:5
   |
13 |     async fn authorize(&mut self, credential: &str) -> Result<()> {
   |     -----^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |     |
   |     `async` because of this
   |
   = note: `async` trait functions are not currently supported
   = note: consider using the `async-trait` crate: https://crates.io/crates/async-trait
   = note: see issue #91611 <https://github.com/rust-lang/rust/issues/91611> for more information

error[E0195]: lifetime parameters or bounds on method `authorize` do not match the trait declaration
  --> ipc_examples/examples/acceptor_auth.rs:13:23
   |
13 |     async fn authorize(&mut self, credential: &str) -> Result<()> {
   |                       ^ lifetimes do not match method in trait

Some errors have detailed explanations: E0195, E0432, E0706.
For more information about an error, try `rustc --explain E0195`.
```

--- a/ipc_examples/Cargo.toml
+++ b/ipc_examples/Cargo.toml
@@ -15,4 +15,5 @@ This module provides examples of using `ipc` feature of `kxkdb` library. The fun
 [dev-dependencies]
 kxkdb={path="../kxkdb", features=["ipc"]}
 chrono="0.4"
+async-trait="0.1"
 tokio={version="1.0", features=["net", "rt", "io-util", "fs", "macros", "rt-multi-thread"]}

